src/heat-map/heat-map.component.ts

```diff
@@ -211,14 +218,14 @@ export class HeatMapComponent extends BaseChartComponent {
   }
 
   setColors(): void {
-    this.colors = new ColorHelper(this.scheme, 'linear', this.valueDomain);
+    this.colors = new ColorHelper(this.scheme, this.scaleType, this.valueDomain);
   }
 
   getLegendOptions() {
     return {
-      scaleType: 'linear',
+      scaleType: this.scaleType,
```

Comment:
Ordinal scaleType here would only make sense if we used a discrete color scale instead of the linear one we currently use. Let's remove it until we add that functionality please.